genesyscloud/idp_adfs/resource_genesyscloud_idp_adfs_schema.go

```diff
@@ -0,0 +1,99 @@
+package idp_adfs
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"terraform-provider-genesyscloud/genesyscloud/provider"
+	resourceExporter "terraform-provider-genesyscloud/genesyscloud/resource_exporter"
+	registrar "terraform-provider-genesyscloud/genesyscloud/resource_register"
+)
+
+/*
+resource_genesycloud_idp_adfs_schema.go holds four functions within it:
+
+1.  The registration code that registers the Datasource, Resource and Exporter for the package.
+2.  The resource schema definitions for the idp_adfs resource.
+3.  The datasource schema definitions for the idp_adfs datasource.
+4.  The resource exporter configuration for the idp_adfs exporter.
+*/
+const resourceName = "genesyscloud_idp_adfs"
+
+// SetRegistrar registers all of the resources, datasources and exporters in the package
+func SetRegistrar(regInstance registrar.Registrar) {
+	regInstance.RegisterResource(resourceName, ResourceIdpAdfs())
+	regInstance.RegisterExporter(resourceName, IdpAdfsExporter())
+}
```

Comment:
You'll need to import this new package `idp_adfs` to main.go, and call this function from there. You'll see how it is done with all the other packages there. After that, run `go generate` from the command line to generate the docs. If nothing changes, that's a good sign.